versions/3.0.md

```diff
@@ -969,7 +989,6 @@ Describes a single response from an API Operation, including design-time, static
 Field Name | Type | Description
 ---|:---:|---
 <a name="responseDescription"></a>description | `string` | **Required.** A short description of the response. [CommonMark syntax](http://spec.commonmark.org/) can be used for rich text representation.
-<a name="responseSchema"></a>schema | [Schema Object](#schemaObject) | A definition of the response structure. It can be a primitive, an array or an object. If this field does not exist, it means no content is returned as part of the response. As an extension to the [Schema Object](#schemaObject), its root `type` value may also be `"file"`. This SHOULD be accompanied by a relevant `produces` media type.
 <a name="responseHeaders"></a>headers | [Headers Object](#headersObject) | A list of headers that are sent with the response.
 <a name="responseExamples"></a>examples | [Example Object](#exampleObject) | An example of the response message.
```

Comment:
This PR does not fix the mention of `produces` (which no longer exists) in [Example Object](https://github.com/OAI/OpenAPI-Specification/blob/650000a84424b3a3c7ae8f375e0774210299daa3/versions/3.0.md#example-object) and [Examples Object](https://github.com/OAI/OpenAPI-Specification/blob/650000a84424b3a3c7ae8f375e0774210299daa3/versions/3.0.md#examples-object)